OG_CATTLE V1

glenjimen opened the pull request:
Replaced original CATTLE1 with OG version.

glenjimen committed:
Delete objects/Vehicle/Train/GOODS3/.gitkeep

glenjimen committed:
Add files via upload

Added OG_CATTLE.dat, blender file and textures.

glenjimen committed:
Delete objects/Vehicle/Train/CATTLE1/.gitkeep

LeftofZen commented on objects/Vehicle/Train/CATTLE1/CATTLE1-body_v1.png:
can you remove the `v1` from all the filenames please, it isn't necessary

LeftofZen reviewed (changes requested):
please remove `v1` from filenames as it isn't necessary (that's what git is for :) )

glenjimen committed:
Add files via upload

Removed creator version numbers.  Sorry, I neglected to clean those up before upload.

glenjimen committed:
Delete objects/Vehicle/Train/CATTLE1/CATTLE1-body_v1.png

glenjimen committed:
Delete objects/Vehicle/Train/CATTLE1/OG_CATTLE1-roof_v1.png

glenjimen committed:
Delete objects/Vehicle/Train/CATTLE1/OG_CATTLE1_v1.blend

LeftofZen committed:
Merge branch 'main' into OG_CATTLE1
